src/utils/constants.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum, auto
+
+
+class ChoicesEnum(Enum):
+    """Augments a regular enum with methods to return lists of enum
```

Comment:
comment unclear

Reply:
Seeing what the class does, do you have a suggestion? Agree that it's a bit short, but it does describe the functionality fine.

Reply:
Extended enum class containig method to list the enum choices.